pkg/clustermanager/kubetest2/gke.go

```diff
@@ -128,69 +126,45 @@ func Run(opts *Options, cc *GKEClusterConfig) error {
 		kubetest2Flags = append(kubetest2Flags, "--project="+cc.GCPProjectID)
 	}
 
-	return createGKEClusterWithRetries(kubetest2Flags, opts, cc)
-}
+	if cc.PrivateClusterAccessLevel != "" {
+		kubetest2Flags = append(kubetest2Flags, "--private-cluster-access-level="+cc.PrivateClusterAccessLevel)
+		kubetest2Flags = append(kubetest2Flags, "--private-cluster-master-ip-range="+strings.Join(cc.PrivateClusterMasterIPSubnetRange, ","))
```

Comment:
Are the network masks not needed? I see in the code removed the `--private-cluster-master-ip-range` seemes to expect the mask?

Reply:
They were needed before to prevent IP conflicts when retrying on the private cluster creations, but now we are switching to the retry feature for kubetest2 gke deployer, which already supports this, so this field can be safely dropped.